website/public/index.ejs.html

```diff
@@ -683,8 +683,7 @@ <h2 id="linter-feature-comparison">Features</h2>
               <td class="good"><span aria-label="stable">✅</span></td>
             </tr>
           </tbody>
-
-          <tbody class="editors hide-in-mobile">
+          <thead class="hide-in-mobile">
```

Comment:
There can be only one thead per table [1], and the thead must not be after a tbody. There's a thead already on line 419, and there's a tbody immediately above this thead. So I don't think this HTML is legal.

[1] https://html.spec.whatwg.org/multipage/tables.html#the-thead-element

Reply:
legal in what sense? It works! you can have thead followed by tbody followed by thead followed by tbody. Maybe its the browser that makes it work even if its not in the spec?

Reply:
> What is the motivation for this change?

I was trying to make the table on quick-lint sortable with firefox plugin. I know you don't care :crying_cat_face:

Reply:
![image](https://github.com/quick-lint/quick-lint-js/assets/49783296/a7f21d22-573f-40f8-92c6-6329af0115f9)
But you really don't!

Reply:
Do I take the first row of each subsequent tbody and make it clickable then?

Reply:
If a `tbody` has only one `tr`, and that `tr` only has `th`s, then I think you can treat that `tbody` as a `thead` for table-sort-js.

Reply:
I think you mean if a `tbodys` first `tr` is only `th` `s` . Why would a tbody only have one tr big oof; this is complicated...